### DISCUSSION TITLE
Rendering HTML tables from Markdown

### DISCUSSION THREAD

**Discussion by Clarity-89** [contentlayerdev/contentlayer#403] [Q&A]
Couldn't find any info on this, but is it possible to generate tables defined inside MDX as Markdown tables? For ex., I have this table in my post and it isn't rendered as the HTML table in the output:
```
| Name | Type | Description |
| ---- | ---- | ----------- |
| `name` | string | Name of the resource |
| `path` | string | Path to the resource |
```

**Comment by aaronlin0122** [comment]
Markdown doesn't support table syntax, but you can use [remark-gfm](https://github.com/remarkjs/remark-gfm) to support it.

To be able to use remark/rehype plugins in Contentlayer, you can read it [here](https://www.contentlayer.dev/docs/reference/source-files/make-source#markdown).

**Reply by Clarity-89** [reply]
In the end I created a separate `Table` React component, which I then use inside the mdx.

**Reply by cpatti97100** [reply]
hi @Clarity-89 can you please share more about your approach?
